packages/next/server/font-utils.ts

```diff
@@ -77,3 +78,58 @@ export function getFontDefinitionFromManifest(
     })?.content || ''
   )
 }
+
+function parseGoogleFontName(css: string): Array<string> {
+  const regex = /font-family: ([^;]*)/g
+  const matches = css.matchAll(regex)
+  const fontNames = new Set<string>()
+
+  for (let font of matches) {
+    const fontFamily = font[1].replace(/^['"]|['"]$/g, '')
+    fontNames.add(fontFamily)
+  }
+
+  return [...fontNames]
+}
+
+function calculateOverrideCSS(font: string, fontMetrics: any) {
+  const fontName = font.toLowerCase().trim().replace(/ /g, '-')
+  const fontKey = font.toLowerCase().trim().replace(/ /g, '')
```

Comment:
What do you think of changing `fontMetrics` to be keyed by hyphen (i.e. `fontName`). Might be nice to avoid having two formatted names of the same font

Reply:
This was the format from the google fonts dataset. It would be hard to match names with hyphens since the data set uses names without hypens